### DISCUSSION TITLE
I imported segmentation node from "Total segmentor", while thing is fine in segment editor, I couldn't make choice of label in smart editor

### DISCUSSION THREAD

**Discussion by linfanluntan** [Project-MONAI/MONAILabel#1767] [Q&A]
any hints? Thanks!

**Comment by diazandr3s** [comment]
Hi @linfanluntan,

Could you please share more details of what the issue is?

**Comment by lassoan** [comment]
Please try with the latest Slicer Preview Release and let us know if you still experience any issues.
